### PR TITLE
Styler and new datasets

### LINKED ISSUE #508: Add styler ignore directives
@mattansb In this [PR](https://github.com/easystats/effectsize/pull/507), you mentioned that you don't like formatting that follows the tidyverse style guide. But, in our [contributing guidelines](https://easystats.github.io/effectsize/CONTRIBUTING.html), we mention that we do follow this style guide. I am not happy that `{effectsize}` is the only exception now.

A way out of this is to ignore parts of code from `{styler}`. This way, we don't need to skip style guide only in one repo.

``` r
styler::style_text("
a=1

x   <- 2
xyz <- 3
")
#> 
#> a <- 1
#> 
#> x <- 2
#> xyz <- 3

styler::style_text("
a=1

# styler: off
x   <- 2
xyz <- 3
# styler: on
")
#> 
#> a <- 1
#> 
#> # styler: off
#> x   <- 2
#> xyz <- 3
#> # styler: on
```

<sup>Created on 2022-10-01 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

For rmarkdown code chunks, you can use `tidy = FALSE`.

Screenshot 2022-10-01 at 06 21 59

For more, see: https://styler.r-lib.org/reference/stylerignore.html?q=ignore

### PR DESCRIPTION
**Pull request by mattansb**
Closes #508

### CODE REVIEW

File: R/cohens_g.R [hunk 1]
```diff
@@ -35,13 +35,22 @@
...
 #'
 #' @examples
-#' Performance <- matrix(c(794, 150,
-#'                         86, 570), nrow = 2)
-#' dimnames(Performance) <- list("1st Survey" = c("Approve", "Disapprove"),
-#'                               "2nd Survey" = c("Approve", "Disapprove"))
-#' Performance
 #'
-#' cohens_g(Performance)
+#' data("Test_results")
```
Comment (IndrajeetPatil) [inline R/cohens_g.R:39]: _Test_results_ is too generic.

Can you think of a more descriptive name? How about _screening_test_ or just _screening_?

File: data-raw/df data.R [hunk 2]
```diff
@@ -0,0 +1,249 @@
+# Test_results ------------------------------------------------------------
+
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:2]:
```suggestion

# styler: off
```

File: data-raw/df data.R [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+  Test1 = rep(Test_results[[1]], times = Test_results$Freq),
+  Test2 = rep(Test_results[[2]], times = Test_results$Freq)
+)
+i <- sample(1600, size = 800)
+Test_results$Diagnosis[i] <- Test_results$Test1[i]
+Test_results$Diagnosis[-i] <- Test_results$Test2[-i]
+Test_results$Diagnosis <- factor(Test_results$Diagnosis, labels = c("Neg", "Pos"))
+
+save(Test_results, file = "data/Test_results.rdata")
+
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:25]:
```suggestion
# styler: on
```

File: data-raw/tabular data.R [hunk 4]
```diff
@@ -0,0 +1,67 @@
+
```
Comment (IndrajeetPatil) [inline data-raw/tabular data.R:1]:
```suggestion
# styler: off
```

File: data-raw/tabular data.R [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+# Food --------------------------------------------------------------------
+
+food_class <- matrix(c(
+  47, 0, 0,
+  0, 12, 21
+), nrow = 2, byrow = TRUE)
+dimnames(food_class) <- list(c("Vegan", "Not-Vegan"), c("Soy", "Milk", "Meat"))
+food_class <- as.table(food_class)
+
+save(food_class, file = "data/food_class.rdata")
```
Comment (IndrajeetPatil) [inline data-raw/tabular data.R:67]:
```suggestion
save(food_class, file = "data/food_class.rdata")
# styler: on
```

File: data-raw/df data.R [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+  Test1 = rep(Test_results[[1]], times = Test_results$Freq),
+  Test2 = rep(Test_results[[2]], times = Test_results$Freq)
+)
+i <- sample(1600, size = 800)
+Test_results$Diagnosis[i] <- Test_results$Test1[i]
+Test_results$Diagnosis[-i] <- Test_results$Test2[-i]
+Test_results$Diagnosis <- factor(Test_results$Diagnosis, labels = c("Neg", "Pos"))
+
+save(Test_results, file = "data/Test_results.rdata")
+# styler: on
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:26]:
```suggestion

```

File: data-raw/df data.R [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+    4, 5, 5, 7, 4, 6, 3, 5, 4, 6, 4, 5, 4, 4, 4, 5, 3, 5, 5,
+    5, 6, 4, 2, 2, 5, 5, 6, 5, 3, 4, 5, 6, 4, 5, 4, 4, 3, 5,
+    5, 5, 4, 3, 4, 4, 5, 3, 3, 4, 5, 4, 5, 4, 5, 5, 5, 4, 5,
+    6, 3, 3, 6, 6, 4, 5, 4, 4, 5
+  )
+)
+
+hardlyworking$is_senior <- hardlyworking$seniority > 4
+
+save(hardlyworking, file = "data/hardlyworking.rdata")
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:250]:
```suggestion
save(hardlyworking, file = "data/hardlyworking.rdata")
# styler: on
```

File: data-raw/df data.R [hunk 8]
```diff
@@ -0,0 +1,250 @@
+# Test_results ------------------------------------------------------------
+
+# styler: off
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:3]:
```suggestion

```

File: data-raw/df data.R [hunk 9]
```diff
@@ -0,0 +1,250 @@
+# Test_results ------------------------------------------------------------
```
Comment (IndrajeetPatil) [inline data-raw/df data.R:1]:
```suggestion
# styler: off

# Test_results ------------------------------------------------------------
```

**Comment by mattansb** [comment]
Hmmm do we need those in the `data-raw` files? Or are you just playing around?

**Comment by IndrajeetPatil** [comment]
We need them there. Otherwise, the next time you run _styler_, it's going to change formatting of those files. 

IINM, that's the formatting you wanted to preserve. Although it is less crucial as it is no longer user-facing code.

**Comment by mattansb** [comment]
No, I only cared about user facing stuff, but now with the new data that seems to be okay

**Pull request merged**